### PR TITLE
DropDownEditor: add input aria-controls attribute (T1127573)

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js [hunk 1]
```diff
@@ -4947,6 +4947,22 @@ QUnit.module('aria accessibility', {}, () => {
...
+
+    QUnit.test('aria-controls should be set if the popup has been rendered', function(assert) {
+        const isDesktop = devices.real().deviceType === 'desktop';
+
+        if(isDesktop) {
+            const $element = $('#dateBox').dxDateBox({ value: new Date() });
+            const $input = $element.find(`.${TEXTEDITOR_INPUT_CLASS}`);
+
+            const $button = $element.find(`.${DROP_DOWN_BUTTON_CLASS}`);
+            $($button).trigger('dxclick');
```
Comment (Zedwag) [inline testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js]: why not just use [open()](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDateBox/Methods/#open) method here? And `field()` to get input element

File: testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js [hunk 2]
```diff
@@ -4947,6 +4947,22 @@ QUnit.module('aria accessibility', {}, () => {
...
+
+            assert.strictEqual(Boolean($input.attr('aria-controls')), true, 'aria-controls attribute has been setted');
+        } else {
+            assert.ok(true, 'skip test on devices');
+        }
+    });
```
Comment (Zedwag) [inline testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js]: let's add test with `deferRendering: false`

File: testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js (hunk 1 above)
Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js]: https://github.com/DevExpress/DevExtreme/pull/23257/commits/bbfe0c9f3536e4c20175608626407cfe740b0cea

File: testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js (hunk 2 above)
Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js]: https://github.com/DevExpress/DevExtreme/pull/23257/commits/bbfe0c9f3536e4c20175608626407cfe740b0cea

**Review by Zedwag — DISMISSED** [review]

**Review by Zedwag — DISMISSED** [review]

**Review by Zedwag — DISMISSED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**